internal/commands/user/inputs.go

```diff
@@ -0,0 +1,110 @@
+package user
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	userDataEmail = "email"
+	userDataName  = "name"
+)
+
+func validAuthProviderTypes() []interface{} {
+	apts := make([]interface{}, 0, len(realm.ValidAuthProviderTypes))
+	for _, apt := range realm.ValidAuthProviderTypes {
+		apts = append(apts, apt)
+	}
+	return apts
+}
+
+func displayUser(apt realm.AuthProviderType, user realm.User) string {
```

Comment:
i think we can simplify this and `displayUserData` methods here a bit.  something like:

```go
func displayUser(apt relam.AuthProviderType, user realm.User) string {
  var sb strings.Builder
  sb.WriteString(apt.Display() + " - ")
  if data := displayUserData(apt, user); user != "" {
    sb.WriteString(data + " - ") // probably also worth moving " - " into a `const` somewhere, maybe terminal.DelimiterInline
  }
  sb.WriteString(user.ID)
  return sb.String()
}

func displayUserData(apt realm.AuthProviderType, user realm.User) string {
  // return the "" if no match on provider type for now
}
```